helm/aladdin-demo/templates/shared/configMap.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
 # Create key - value entries in the data map that other files can look up
 # Load them into the environment by using configMapRef in the deploy.yaml file
 data:
+  PROJECT_NAME: {{ .Chart.Name }}
```

Comment:
Inject PROJECT_NAME (in this case, aladdin-demo).

Doing things this way makes it easier for people basing new services on this project to change the name without breaking service discovery.

Reply:
noice